indra/llcommon/llerror.h

```diff
@@ -308,15 +308,23 @@ namespace LLError
     class LLUserWarningMsg
     {
     public:
-        typedef std::function<void(const std::string&, const std::string&)> Handler;
+        typedef enum
+        {
+            ERROR_OTHER = 0,
+            ERROR_BAD_ALLOC = 1,
+            ERROR_MISSING_FILES = 2,
+        } eLastExecEvent;
```

Comment:
Might be good to add a comment something like: "These are used to select `LAST_EXEC_OTHER_CRASH`, `LAST_EXEC_BAD_ALLOC` and `LAST_EXEC_MISSING_FILES`, respectively, which aren't visible from here."

Reply:
Agree, will keep it in mind if I get to continue.

Reply:
I probably should have done somthing like ERROR_BAD_ALLOC = 6 instead of using switch.